docs/v13/views/support.html

```diff
@@ -22,25 +22,14 @@
 
      <h1 class="govuk-heading-xl">Support</h1>
 
-   <p>If you've got a question, idea or suggestion, get in touch with the GOV.UK Prototype team.</p>
+	<p>The GOV.UK Prototype Kit is currently maintained by the [GOV.UK Design System team](https://design-system.service.gov.uk/get-in-touch/). There is only [minimal support in place](https://github.com/alphagov/govuk-prototype-kit/issues/2389) as the current version is stable.</p>
```

Comment:
I need to double check but I'm pretty sure markdown links won't work here – they'll need to be HTML links instead (like the Slack link in the next paragraph).